packages/components/organisms/f-header/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-header",
   "description": "Fozzie Header – Globalised Header Component",
-  "version": "4.11.1",
+  "version": "4.12.0",
```

Comment:
is it a correct version upd for a typo fix? or should it be `4.11.2` instead?

Reply:
Think `4.11.2` would make sense here.